WordPress/gutenberg
# [RNMobile] Prevent bold headings after autocorrection

## Description
Related PRs:
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/4557
- https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1334

## Testing Instructions
See Gutenberg Mobile PR for testing instructions:
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/4557

## Screenshots 

## Types of changes




Bug fix

## Checklist:
